tests/unit/core/test_installAddon.js

```diff
@@ -1151,7 +1151,7 @@ describe(__filename, () => {
         });
       });
 
-      it('passes an undefined hash when installURL is not found', () => {
+      it('rejects when installURL is not found', () => {
```

Comment:
I'm now sure how this is related to the `src` issue?

Reply:
That is because we use Flow and it is sometimes annoying. In this case, Flow revealed the following error: it was not possible to pass an undefined `installURL`. So I had to fix this too.. It was revealed because a function used somewhere under the hood got Flow annotations.